Lombiq.TrainingDemo/Controllers/YourFirstOrchardCoreController.cs

```diff
@@ -17,6 +17,7 @@
 
 namespace Lombiq.TrainingDemo.Controllers;
 
+[Route("[controller]/{action=Index}")]
```

Comment:
Neither of the actions can be accessed by their documented/configured URLs anymore.